lib/honeybee/model_object.rb

```diff
@@ -116,6 +132,21 @@ def self.clean_identifier(str)
       encode_str.gsub(/[^.A-Za-z0-9_-]/, '_').gsub(' ', '_')
     end
 
+    # Create methods to get and set display name
+    if !OpenStudio::Model::ModelObject.method_defined?(:displayName)
```

Comment:
I just wanted to say that I really like the way this is implemented. It's very clean and will be much easier to maintain over time!